user-interface/src/data-verification/consolidation/ConsolidationOrderAccordion.test.tsx

```diff
@@ -102,11 +58,18 @@ describe('ConsolidationOrderAccordion tests', () => {
       'consolidations-enabled': true,
     };
     vitest.spyOn(FeatureFlagHook, 'default').mockReturnValue(mockFeatureFlags);
-    setupApiGetMock();
+    vitest
```

Comment:
I've noticed that some files have `vitest.spyOn` rather than `vi.spyOn` which generally leads to new spies being written with vitest. We should probably change them all. This could be in a different PR.